product_docs/docs/edbcloud/beta/getting_started/03_create_cluster.mdx

```diff
@@ -5,35 +5,68 @@ title: "Creating a Cluster"
+3. Select the region where you want your cluster to be deployed in the **Region** field. For the best performance, EDB typically recommends that this region should be the same as other resources you have that communicate with your cluster.
+4. In the the **Instance Type** section, select the number of vCPUs and amount of memory you desire.
+5. In the **Storage** section, select **Volume Type**, and in **Volume Properties** the type and amount of storage needed for your cluster.
    !!! Note
        EDB Cloud currently supports Azure Premium SSD storage types. See [the Azure documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#premium-ssd) for more information.
-12. (Optional) Expand the **Advanced Customization** section, to specify:
-    1.  The desired amount of connections your cluster allows for other applications. You can either enter the amount or use the arrows. The default is 100.
-   
-    1.  Whether to use private or public networking using the **Networking** slide button. Networking is set to Private by default. Private networking allows only IP addresses within your private network to connect to your cluster. Public means that any client can connect to your cluster’s public IP address over the internet.
-17. Select **Create Cluster**. It might take a few minutes to deploy.
+6. In the **Networking** section, Whether to use private or public networking using the **Networking** slide button. Networking is set to Public by default. Public means that any client can connect to your cluster’s public IP address over the internet.
+Private networking allows only IP addresses within your private network to connect to your cluster.
+7. To optionally make updates to your database configuration parameters, select **Next: DB Configuration**.
+
+## DB Configuration
+In the **Parameters** section, you can update the value of the database configuration parameters, as needed. These parameters are also referred to as GUC variables, see [What Is a GUC Variable?](https://www.enterprisedb.com/blog/what-guc-variable) for more information.
+
+The list of parameters is populated based on the type of database you selected on the **Operational Settings** tab. For more information about the parameters for your database type, see:
+
+* [Setting Parameters](https://www.postgresql.org/docs/current/config-setting.html) and [Server Configuration](https://www.postgresql.org/docs/current/runtime-config.html) topics in the PostreSQL documentation.
+* [Summary of Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/02_summary_of_configuration_parameters/#summary_of_configuration_parameters) and [Configuration Parameters](https://www.enterprisedb.com/docs/epas/latest/epas_guide/03_database_administration/01_configuration_parameters/) topics in the EDB Postgres Advanced Server documentation.
```

Comment:
Does anyone object to linking to https://postgresqlco.nf/ here? It's run by OnGres / Alvaro Hernandez. I think it's a great resource for users.

/cc @theadamwright

Reply:
Is that resource just for Postgres or do the tools and doc also apply to EDB Postgres Advanced Server? I'm assuming the former.